test/functional/core/job_spec.lua

```diff
@@ -979,6 +981,7 @@ describe('jobs', function()
       nvim('command', 'let g:job_opts.pty = 1')
       nvim('command', 'let exec = [expand("<cfile>:p")]')
       nvim('command', "let j = jobstart(exec, g:job_opts)")
+      j = eval'j'
```

Comment:
It's really just a smaaaall nitpick, but I think we should use explicit parentheses here. :>

Reply:
one day even you will learn to appreciate the full power of free form lua syntax.